### PR TITLE
 RHICOMPL-220 - Inconsistency in Compliance severity capitalization 

### PR DESCRIPTION
**Pull request by dLobatog**
Depends on https://github.com/RedHatInsights/frontend-components/pull/71 because otherwise the page would not even load because of the GraphQL changes.

### CODE REVIEW

File: packages/inventory-compliance/src/SystemRulesTable.js [hunk 1]
```diff
@@ -180,7 +180,7 @@ class SystemRulesTable extends React.Component {
         cells: [
             title,
             profile.name,
-            severity,
+            severity.charAt(0).toUpperCase() + severity.slice(1),
```
Comment (akofink) [inline packages/inventory-compliance/src/SystemRulesTable.js]: Why not just make it all lowercase? Or, even if you want it capitalized, why not apply a CSS class with `text-transform: capitalize;`? If it's just for the purposes of presentation, I think the CSS style makes more sense.

**Comment by akofink** [comment]
#71 is merged. Please rebase

File: packages/inventory-compliance/src/SystemRulesTable.js [hunk 2]
```diff
@@ -457,6 +457,7 @@ class SystemRulesTable extends React.Component {
                         />
                     </TableToolbar>
                     <Table
+                        className='compliance-rules-table'
```
Comment (dLobatog) [inline packages/inventory-compliance/src/SystemRulesTable.js]: I've changed the capitalization to happen through CSS here

File: packages/inventory-compliance/src/SystemRulesTable.js [hunk 3]
```diff
@@ -312,7 +312,7 @@ class SystemRulesTable extends React.Component {
     filterByPolicy = (policy, rows) => {
         const filteredRows = [];
         rows.forEach((row, i) => {
-            if (row.hasOwnProperty('isOpen') && policy.includes(row.cells[POLICY_COLUMN])) {
+            if (row.hasOwnProperty('isOpen') && policy.includes(row.cells[POLICY_COLUMN].toLowerCase())) {
```
Comment (dLobatog) [inline packages/inventory-compliance/src/SystemRulesTable.js]: There was a bug here - when a report contained capitalized `severity` fields, they were not filtered properly with this. The filter was checking exclusive for lowercase: https://github.com/RedHatInsights/frontend-components/blob/master/packages/inventory-compliance/src/RulesComplianceFilter.js#L21 - so we can convert the severity here to lowercase to fix the bug.

**Pull request merged**